pkg/pillar/cmd/zedrouter/flowstats.go

```diff
@@ -164,10 +174,11 @@ func FlowStatsCollect(ctx *zedrouterContext) {
 
 			var sequence, flowIdx int
 
+			// fill in the partial scope information, later the aclNum and aclAttr will decide
+			// if we have a match in this flow into app/bridge scope
 			scope := types.FlowScope{
 				UUID:      instData.appNet[appIdx],
-				Intf:      bnx,
-				Localintf: instData.bnNet[bnx].vif,
+				Localintf: instData.bnNet[bnx].bridge,
```

Comment:
Is Localintf not the app vif?

Reply:
I asked Bharani, he said 'Intf' is the app intf name such as 'eth0' or 'indirect'; and 'Localintf' is the bridgename, such as 'bn1'. thus to conform to that.